src/js/models/layer.js

```diff
@@ -469,6 +559,26 @@ define(function (require, exports, module) {
     var _extractHasLayerEffect = function (layerDescriptor) {
         return !!object.getPath(layerDescriptor, "layerEffects");
     };
+    
+    /**
+     * Return the capitalized layer kind name. For example: SMARTOBJECT -> SmartObject
+     *
+     * @private
+     * @param {LayerKind} layerKind
+     * @return {string}
+     */
+    var _capitalizeLayerKindName = function (layerKind) {
```

Comment:
This function should be called camelCaseLayerKindName, as you're camelCasing it.